migrations/create_webmaker_db.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS "users"
```

Comment:
In an additional table? Yeah, but I think that'll be a separate ticket, figuring out how that all works...